src/inspector_agent.cc

```diff
@@ -592,6 +612,16 @@ class NodeInspectorClient : public V8InspectorClient {
     return env_->isolate_data()->platform()->CurrentClockTimeMillis();
   }
 
+  std::unique_ptr<StringBuffer> resourceNameToUrl(
+      const StringView& resource_name_view) override {
+    std::string resource_name =
+        protocol::StringUtil::StringViewToUtf8(resource_name_view);
+    if (!IsFilePath(resource_name))
+      return nullptr;
+    node::url::URL url = node::url::URL::FromFilePath(resource_name);
+    return Utf8ToStringView(url.protocol() + "//" + url.path());
```

Comment:
Hmm, so our URL class doesn't actually have a `href()` getter. That seems odd. Could you put a TODO here and file a bug?

Reply:
added todo, created bug: https://github.com/nodejs/node/issues/22610